<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OwnerRepairEvalWidget</class>
 <widget class="QDialog" name="OwnerRepairEvalWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>报修评价</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>360</width>
     <height>286</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>报修评价</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelRepairOrderId">
      <property name="text">
       <string>报修单ID：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxRepairOrderId">
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="placeholderText" stdset="0">
       <string>请输入报修单ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>评分</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="labelEvaluation">
      <property name="text">
       <string>评价内容：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditEvaluation"/>
    </item>
    <item>
     <widget class="QLabel" name="labelWordLimit">
      <property name="text">
       <string>请填写评价，500字内</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSubmitEvaluation">
      <property name="text">
       <string>提交评价</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
